planning/behavior_path_planner/src/scene_module/lane_change/util.cpp

```diff
@@ -374,6 +374,9 @@ LaneChangePaths getLaneChangePaths(
     const auto target_lane_reference_path = getReferencePathFromTargetLane(
       route_handler, target_lanelets, lane_changing_start_pose, target_lane_length, lc_dist,
       required_total_min_distance, forward_path_length, resample_interval, is_goal_in_route);
+    if (target_lane_reference_path.points.empty()) {
+      return candidate_paths;
```

Comment:
`continue` is also suitable in this case.
It ensure all deceleration value is considered. 

What do you think about this?

Reply:
Thanks, exactly!
I fixed it.
[d5a1f96](https://github.com/autowarefoundation/autoware.universe/pull/2752/commits/d5a1f969691c0087d52dd9cf4db2b7b5aeb23e74)